ministryofjustice/laa-apply-for-criminal-legal-aid
# [CRIMAPP-719] Update other benefit details content

## Description of change
Content updates to other benefit details row

## Link to relevant ticket
[CRIMAPP-719](https://dsdmoj.atlassian.net/browse/CRIMAPP-719)

## Notes for reviewer

## Screenshots of changes (if applicable)

### Before changes:

### After changes:
<img width="1280" alt="Screenshot 2024-04-15 at 17 12 17" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/52556e0e-6c49-4fe3-9367-18356470fd9b">

## How to manually test the feature
Start a means tested application
Fill in details until you get to the income benefit question in the income assessment section
Select the other benefit option and fill in the amount, frequency and other details information 
Fill in the rest of the application
On the cya, review and completed application screens you should see the rows for other income benefits with the correct content


[CRIMAPP-719]: https://dsdmoj.atlassian.net/browse/CRIMAPP-719?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
